Source/WTF/Scripts/Preferences/UnifiedWebPreferences.yaml

```diff
@@ -7899,6 +7899,19 @@ WebSocketEnabled:
       "PLATFORM(WATCHOS)": false
       default: true
 
+WebTransportEnabled:
+  type: bool
+  status: unstable
```

Comment:
Note that I think you'll need "testable" for it to get enabled in layout tests iirc.